File: spring-boot-oauth2-auth-server/src/main/java/com/bittokazi/oauth2/auth/server/app/models/base/Oauth2Response.java
package com.bittokazi.oauth2.auth.server.app.models.base;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class Oauth2Response {

    private String access_token;

    private String refresh_token;

    private String token_type;

    private Long expires_in;

}

File: spring-boot-oauth2-auth-server/src/main/kotlin/com/bittokazi/oauth2/auth/server/app/models/base/Oauth2Response.kt
package com.bittokazi.oauth2.auth.server.app.models.base

import lombok.Getter
import lombok.Setter

@Getter
@Setter
class Oauth2Response {
    lateinit var access_token: String

    lateinit var refresh_token: String

    lateinit var token_type: String

    var expires_in: Long = 0
}
